Sources/SwiftSyntaxMacros/MacroSystem.swift

```diff
@@ -468,6 +476,40 @@ extension MacroApplication {
   }
 }
 
+// FIXME: This was copied from 'SwiftSyntaxMacroExpansion' module.
+// All expansion logics should be unified.
```

Comment:
Can’t you make this `@_spi(something) public` and then use it from `SwiftSyntaxMacroExpansion`. `SwiftSyntaxMacroExpansion` already depends on `SwiftSyntaxMacros` AFAICT.